### ISSUE TITLE
Badge on Explorer is confusing

### ISSUE THREAD

**Issue by zxch3n** [Gruntfuggly/todo-tree#681]
Type: <b>Bug</b>

Currently, the badge is on the file explorer icon. 

<img width="58" alt="image" src="https://user-images.githubusercontent.com/18425020/195770762-99d9a1b2-9904-4abe-bea3-166a2f5e7604.png">

Is it intentional or a bug? It's very confusing to have a number on this icon. I thought this was a VSCode issue. Only after bisecting I found out this extension causes it.

Extension version: 0.0.216
VS Code version: Code 1.72.2 (d045a5eda657f4d7b676dedbfa7aab8207f8a075, 2022-10-12T22:15:55.763Z)
OS version: Darwin arm64 21.6.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|6, 4, 3|
|Memory (System)|16.00GB (0.09GB free)|
|Process Argv|--crash-reporter-id d4200526-e225-4483-b5e2-ee423130839d|
|Screen Reader|yes|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vscaac:30438847
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
2144e591:30553903
pyindex848cf:30577861
nodejswelcome1cf:30587006
fc301958:30573243

```

</details>

**Comment by PSAGChristian** [comment]
I agree - this feature shouldn't be activated by default.
Might be nice to have such a featrue it, but it is very confusing because you don't expect this extension to add these badge numbers.

**Comment by zjlovezj** [comment]
I just found a workaround.
To avoid showing badge on Explorer icon, you can drag "TODOS: TREE" from "EXPLORER" / "Primary Side Bar" to "Side Bar".
After that, the badge will disappear from Explorer icon and display on TODOs.

<img width="388" alt="image" src="https://user-images.githubusercontent.com/388222/195809662-f1471210-a0e0-4bc0-82b3-096dc13ab67d.png">

**Comment by Gruntfuggly** [comment]
I didn't realise it would put the badge on the Explore icon too. That's stupid.

You can also disable the badge in the todo tree settings (todo-tree.general.showActivityBarBadge).

I'll see if there is a way to determine if the tree view is in the explorer pane or not.

**Comment by Gruntfuggly** [comment]
I've published a new version 0.0.219, which defaults the badge to off. I've put a note in the README about putting the badge on the Explorer icon. I haven't found a way to determine if the view is in the Explorer pane yet. 😞

**Comment by mustafaabobakr** [comment]
<img width="1087" alt="image" src="https://user-images.githubusercontent.com/27288406/195976721-92c4e4c6-e4eb-4f63-b353-9d07d35dca74.png">
